e2e/test/iothub/command/CommandE2ETests.cs

```diff
@@ -0,0 +1,238 @@
+﻿// Copyright (c) Microsoft. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+#if !NET451
+using System;
+using System.Diagnostics.Tracing;
+using System.Net;
+using System.Text;
+using System.Threading.Tasks;
+using Microsoft.Azure.Devices.Client;
+using Microsoft.Azure.Devices.Common.Exceptions;
+using Microsoft.Azure.Devices.E2ETests.Helpers;
+using Microsoft.VisualStudio.TestTools.UnitTesting;
+
+namespace Microsoft.Azure.Devices.E2ETests.Commands
+{
+    public static class ServiceCommandRequestAssertion
+    {
+        public static int a => 123;
+    }
+
+    public class ServiceCommandRequestObject
+    {
+        public int a { get; set; }
+    }
+
+    public class DeviceCommandResponse
+    {
+        public string name => "e2e_test";
+    }
+
+    [TestClass]
+    [TestCategory("E2E")]
+    [TestCategory("IoTHub")]
+    public class CommandE2ETests : E2EMsTestBase
+    {
+        public const string DeviceResponseJson = "{\"name\":\"e2e_test\"}";
```

Comment:
Q - instead of using json string here, could we deal with objects on both sides -> service and device flow?